### PR TITLE
Hiding layouts via preferences

### LINKED ISSUE #166: Windowbuilder Basic UI Layouts
We require that certain layouts be hidden within the Windowbuilder Basic UI. The only layouts that should be available for the user to use in the WB Basic UI are: Absolute Layout, Box Layout, Grid Layout and GridBag Layout.

This should be reflected in the Layouts dropdown option of the properties view.

### PR DESCRIPTION
**Pull request by MarcelS4**
Closes #166 
image

### CODE REVIEW

File: org.eclipse.wb.core.java/src/org/eclipse/wb/internal/core/model/description/helpers/LayoutDescriptionHelper.java [hunk 1]
```diff
@@ -13,11 +13,13 @@
 import com.google.common.collect.Lists;
```
Comment (wimjongman) [inline org.eclipse.wb.core.java/src/org/eclipse/wb/internal/core/model/description/helpers/LayoutDescriptionHelper.java:13]: Please fix copyrights and add your contribution:

File: org.eclipse.wb.swing/src/org/eclipse/wb/internal/swing/model/layout/LayoutInfo.java [hunk 2]
```diff
@@ -1,13 +1,5 @@
...
- * Copyright (c) 2011 Google, Inc.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- *
- * Contributors:
- *    Google, Inc. - initial API and implementation
- *******************************************************************************/
+/*Copyright(c)2011 Google,Inc.*All rights reserved.This program and the accompanying materials*are made available under the terms of the Eclipse Public License v1.0*which accompanies this distribution,and is available at*http://www.eclipse.org/legal/epl-v10.html
```
Comment (wimjongman) [inline org.eclipse.wb.swing/src/org/eclipse/wb/internal/swing/model/layout/LayoutInfo.java:1]: What happened with the copyright?

**Review by wimjongman — COMMENTED** [review]
Thank you, Marcel. Again some minor comments,

**Comment by wimjongman** [comment]
Marcel, how are the layouts listed without your patch? Is your patch only valid for Swing?

**Comment by vogella** [comment]
@MarcelS4 can you fix the conflicts and update this PR?

**Comment by MarcelS4** [comment]
Closed in favour of #182

**Pull request closed without merging**